common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -5,35 +5,40 @@
 from django.utils import timezone
 
 from openedx.features.smart_referral.models import SmartReferral
-from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_emails
 
 CONTACT_EMAIL = 'contact_email'
 DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
 
 
 class Command(BaseCommand):
     help = """
-    Send follow up referral email to all those contacts
+    Send follow-up referral email to all those contacts
```

Comment:
Please add single indent in line 16.

Reply:
Plus, the docstring is not explanatory enough. We need to better define 'all those contacts'

Maybe we can rephrase it to:

'Send follow-up referral email to all pending contacts'

Reply:
done